src/tests/nurse/test_views.py

```diff
@@ -248,6 +249,26 @@ def test_create_minimal_patient(self, client):
         assert patient.ss_provider_code == ""
         assert patient.birthday is None
 
+    @override_settings(FREE_PATIENT_LIMIT=2)
+    def test_non_subscribed_user_cannot_exceed_patient_limit(self, user, client):
+        """
+        Ensure that a non-subscribed
+        user cannot create more patients than the allowed limit.
+        """
+        LIMIT = settings.FREE_PATIENT_LIMIT
+        assert Patient.objects.count() == 0
+        data = {
+            "firstname": "John",
+            "lastname": "Leen",
+        }
+        for _ in range(LIMIT):
+            response = client.post(self.url, data, format="json")
+            assert response.status_code == status.HTTP_201_CREATED
+        assert Patient.objects.count() == LIMIT
+        response = client.post(self.url, data, format="json")
+        assert response.status_code == status.HTTP_403_FORBIDDEN
+        assert response.data == {"detail": "You have reached the limit of free."}
```

Comment:
Well done :clap: 
Maybe make that `You have reached the limit of free.` message a constant so you can import it in your tests

Reply:
Done